crates/bevy_pbr/src/deferred/pbr_deferred_functions.wgsl

```diff
@@ -106,10 +106,12 @@ fn pbr_input_from_deferred_gbuffer(frag_coord: vec4<f32>, gbuffer: vec4<u32>) ->
 fn deferred_output(in: VertexOutput, pbr_input: PbrInput) -> FragmentOutput {
     var out: FragmentOutput;
 
+#ifdef DEFERRED_PREPASS
```

Comment:
i'm not sure what circumstance would cause you to be calling `deferred_output` when `DEFERRED_PREPASS` isn't set. it seems like it would kind of work as long as you don't need any of the vertex outputs apart from normal and motion vecs, but it's definitely not intended.